examples/static/.gitignore

```diff
@@ -1,3 +1 @@
-*.wasm
-*.js
-snippets/
+*
```

Comment:
`index.html` should not be gitignored

Reply:
should not be commited. `wasm-pack` overwrites `.gitignore` file when building.
`build.sh` script has workaround restore correct .gitignore